test/units/modules/cloud/kubevirt/test_kubevirt_definition.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from units.compat.mock import patch, MagicMock
+
+from ansible.module_utils.k8s.common import K8sAnsibleMixin
+from ansible.module_utils.k8s.raw import KubernetesRawModule
+
+openshift = pytest.importorskip("openshift", minversion="0.6.2")
```

Comment:
Instead of importing `openshift`, have you considered mocking the import instead?

Take a look at https://github.com/ansible/ansible/pull/50850 for an example of how this can be done.

Reply:
Would be much better approach. But we have integration tests with real world scenarios, so I think these unittest are good enough.